docs/src/single-pool.mdx

```diff
@@ -0,0 +1,477 @@
+$ solana config set --keypair usb://ledger/
+```
+
+`spl-single-pool` generally uses the default keypair as the fee-payer,
+the wallet to draw funds from (for instance, to fund new stake accounts),
+and the signing authority on accounts that require one.
+When token accounts are required, it defaults to the default keypair's associated account.
+All of these roles can be overrided by command-line flags.
+
+  </TabItem>
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+#### pnpm
+```console
+pnpm install XXX TODO
+```
+#### Yarn
+```console
+yarn add XXX TODO
+```
+#### npm
+```console
+npm install XXX TODO
+```
+
+### Configuration
+You can connect to different clusters using `Connection` in `@solana/web3.js`
+
+```typescript
+import { Connection, clusterApiUrl } from '@solana/web3.js';
+const connection = new Connection(clusterApiUrl('devnet'), 'confirmed');
+```
+
+### Keypair
+You can either get your keypair using [`Keypair`](https://solana-labs.github.io/solana-web3.js/classes/Keypair.html) from `@solana/web3.js`, or let the user's wallet handle the keypair and use `sendTransaction` from [`wallet-adapter`](https://github.com/solana-labs/wallet-adapter)
+
+  </TabItem>
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+#### pnpm
+```console
+pnpm install XXX TODO
+```
+#### Yarn
+```console
+yarn add XXX TODO
+```
+#### npm
+```console
+npm install XXX TODO
+```
+
+### Configuration
+You can connect to different clusters using `createDefaultRpcTransport` and `createSolanaRpc` in `@solana/web3.js`
+
+```typescript
+import { createDefaultRpcTransport, createSolanaRpc } from '@solana/web3.js';
+const transport = createDefaultRpcTransport({ url: 'https://api.devnet.solana.com' });
+const rpc = createSolanaRpc({ transport });
+```
+
+### Keypair
+Web3.js Next uses [`crypto.subtle`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle) for key management. For local development, one can create an object `{ privateKey, publicKey }` using `crypto.subtle.importKey` to produce signing (private) and verifying (public) keys. Generally, however, one should allow the user wallet to manage keys, as the system when used properly is designed to not allow key material to leave secure storage.
+
+  </TabItem>
+</Tabs>
+
+### Setting up a single-validator pool
+
+#### Creating the pool
+
+A single-validator stake pool can be created permissionlessly, by anyone, for a
+given vote account. This allows you to receive the full staking yield you would by
+staking directly while holding the value in a tokenized form.
+It also allows you to buy or sell stakes smaller than the minimum delegation on the market.
+
+Assuming a vote account `Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC` exists, we create a pool at address `DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb`:
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool initialize Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC
```

Comment:
nit-up-to-you-for-consistency: the earlier examples start the line with `$`, and this one doesn't. maybe it's best to be consistent and not use them anywhere, and instead have an empty line between the command and the output

Reply:
will add the $ actually, ty for catching this, i copied the above stuff from the token docs